src/super_gradients/recipes/variable_setup.yaml

```diff
@@ -0,0 +1,17 @@
+lr:
```

Comment:
Could you please add a comment to each shortcut telling to which "main" parameter it aliases to?

Reply:
sure, added in latest commit + more docs.